client/signup/steps/plans/index.jsx

```diff
@@ -16,6 +16,7 @@ var productsList = require( 'lib/products-list' )(),
 	PlanList = require( 'components/plans/plan-list' ),
 	PlansCompare = require( 'components/plans/plans-compare' ),
 	SignupActions = require( 'lib/signup/actions' ),
+	SkipStepButton = require( 'signup/skip-step-button' ),
```

Comment:
Could we use the skip feature that is available in the [`<StepWrapper />` component](https://github.com/Automattic/wp-calypso/blob/08d21c703334810e43bedf128f69842313322fe6/client/signup/step-wrapper/index.jsx)?